rolemanager/module.py

```diff
@@ -0,0 +1,119 @@
+from typing import List
+
+import discord
+from discord.ext import commands
+
+from pie import utils, check, i18n
+from pie.utils.objects import ConfirmView, ScrollableEmbed
+
+_ = i18n.Translator("modules/fsi").translate
+
+
+# HELPER FUNCTIONS
+class RoleManager(commands.Cog):
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    def _create_embeds(ctx, title: str, description: str) -> List[discord.Embed]:
+        elements = []
+        """Create embed for member list.
+        Args:
+            ctx: Command context.
+            option: Item's title.
+            description: list of items.
+        Returns: :class:`discord.Embed` information embed
+        """
+        chunked_list = list()
+        chunk_size = 15
+
+        for i in range(0, len(description), chunk_size):
+
+            page = utils.discord.create_embed(
+                author=ctx.author,
+                title=title,
+                description="\n".join(description[i : i + chunk_size]),
+            )
+
+            chunked_list.append(description[i : i + chunk_size])
+
+            elements.append(page)
+
+        return elements
+
+    # MAIN
+    @commands.guild_only()
+    @commands.group(name="rolemanager")
+    @check.acl2(check.ACLevel.MOD)
+    async def rolemanager_(self, ctx):
+        """
+        Preview and remove selected roles from members with specified based role.
+
+        preview [base_role] [role_to_remove]
```

Comment:
This is not necessary, help command for subcommands will show it automatically.

Reply:
Comment was removed.

Reply:
You've removed the wrong part of comment. 

```Preview and remove selected roles from members with specified based role.```

This part was OK, the part `preview [base_role] [role_tom_remove]` and the other one is wrong.

Reply:
Ok removed